layouts/index.hbs

```diff
@@ -51,12 +51,8 @@
                         &#47; <a href="https://github.com/nodejs/node/blob/{{ project.currentVersions.stable }}/CHANGELOG.md">{{ labels.stable }}</a>
                     </li>
                     <li>
-                      {{#if project.currentVersions.stable}}
```

Comment:
This if statement was put into place *before* we had Stable and LTS on the website - as that's old news, it could be removed.